tests/transaction_log_parsing.cpp

```diff
@@ -1727,6 +1758,31 @@ TEST_CASE("Transaction log parsing: changeset calcuation") {
             REQUIRE(changes.modified(0, 1));
         }
 
+        SECTION("modifying a subtable in an observed table does not produce a notification") {
+            Row r = target->get(0);
+            auto changes = observe({r}, [&] {
+                target->insert_column(0, type_Table, "subtable");
+                auto subdesc = target->get_subdescriptor(0);
+                subdesc->insert_column(0, type_Int, "value");
+                r.get_subtable(0)->add_empty_row();
+            });
+            REQUIRE_FALSE(changes.modified(0, 0));
+            REQUIRE_FALSE(changes.modified(0, 1));
+        }
+
+        SECTION("modifying a subtable in an observed table does not interfere with notifications for other columns") {
+            Row r = target->get(0);
+            auto changes = observe({r}, [&] {
+                target->insert_column(0, type_Table, "subtable");
+                auto subdesc = target->get_subdescriptor(0);
+                subdesc->insert_column(0, type_Int, "value");
```

Comment:
Sorry, my mistake (that method was removed in core 0.92.0 when the `insert_*` family was removed.